Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1500 @@ label mas_wrs_twitch:
+            eventlabel="mas_wrs_sfm",
+            category=['sourcefilmmaker'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_sfm:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Making a video, [player]?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_sfm')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_unreal",
+            category=['unrealengine'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_unreal:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Are you designing your own game? Very cool!"
+           "This looks very complex. You must be a very smart cookie to have figured all this out, ahaha!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_unreal')
+    return    
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_dos",
+            category=['dosbox'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_dos:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Running an old OS in a modern OS? Pretty neat!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_dos')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_virtualbox",
+            category=['virtualbox'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_virtualbox:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Hmmm, you use virtual machines, [player]? Neat!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_virtualbox')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_gimp",
```

Comment:
this can be removed in favour of the prior mention. Merging as an overarching image editor react.